data/obs-packaging/cc-oci-runtime/update_runtime.sh

```diff
@@ -42,19 +43,23 @@ sed "s/@HASH_TAG@/$hash_tag/g;" update_commit_id.patch-template > update_commit_
 # Update and package OBS
 if [ "$OBS_PUSH" = true ]
 then
-    osc co home:clearlinux:preview:clear-containers-staging/cc-oci-runtime
+    temp=`basename $0`
```

Comment:
Not actually a problem, but I'd use `$( ... )` rather than `` `...` `` as it is much clearer (atleast it is to my eyes ;)